Client/mods/deathmatch/logic/CClientModel.cpp

```diff
@@ -37,7 +37,17 @@ bool CClientModel::Allocate(void)
     // Allocate only on free IDs
     if (!pModelInfo->IsValid())
     {
-        pModelInfo->MakePedModel("PSYCHO");
+        switch (m_eModelType)
+        {
+            case eClientModelType::PED:
+                pModelInfo->MakePedModel("PSYCHO");
+                break;
+            case eClientModelType::OBJECT:
+                pModelInfo->MakeObjectModel(1337);
+                break;
+            default:
+                return false;
+        }
```

Comment:
```suggestion
        pModelInfo->MakeModel(m_eModelType);
```
Split this code into a new function so the method becomes shorter.

Reply:
I think i will add more arguments for MakeObjectModel and MakeVehicleModel and MakeTimedObjectModel later

Reply:
Oh, I just realized the enum is `eClientModelType`. In this case, you can create the function in CClientModel. 

```
bool CClientModel::MakeModel()
{
    switch (m_eModelType)
    {
        case eClientModelType::PED:
            pModelInfo->MakePedModel("PSYCHO");
            break;
        case eClientModelType::OBJECT:
            pModelInfo->MakeObjectModel(1337);
            break;
        default:
            return false;
    }
    return true;
}
```
> I think i will add more arguments for MakeObjectModel and MakeVehicleModel and MakeTimedObjectModel later

Sure, but please don't forget to split the code.

Reply:
Yes, but when we define API for `engineRequestModel`